common/logisticspipes/utils/string/StringUtils.java

```diff
@@ -6,6 +6,8 @@
 import java.util.Iterator;
 import java.util.List;
 
+import javax.xml.soap.Text;
```

Comment:
remove that

Reply:
Don't know how I added that but removed it now